.github/workflows/prc.yml

```
@@ -31,6 +31,38 @@ jobs:
       - name: Build
         run: go build -v ./...
+  chromadb-integration:
+    name: chromadb 1.5.9 integration
+    runs-on: ubuntu-latest
+    services:
+      chromadb:
+        image: chromadb/chroma:1.5.9
+        ports:
+          - 8000:8000
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)